docs/lib/datastore/auth-model.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Models with authorization
+description: Learn more about how DataStore handles Models with finer grain authorization access checks.
+---
+
+DataStore has the capability to operate on models with finer grain authorization checks. In GraphQL, this is done with the `@auth` directive defined in the [GraphQL Transformer documentation](~/cli/graphql-transformer/directives.md#auth).
+
+<amplify-callout>
+
+**Note:** this API is under development and it is not generally available yet.
+
+</amplify-callout>
```

Comment:
If you want this to show up for only iOS, i believe this content also needs to be in a fragment with platform="ios"

Reply:
I see.. those pages were empty ( and i think we were including this page in menu.json) so i thought maybe this was a mistake.. but menu.json seems to be no longer updated, so this makes sense.